### ISSUE TITLE
Newrly added device is "disabled" by default

### ISSUE THREAD

**Issue by SeLLeRoNe** [home-assistant/home-assistant-android#263]
**Home Assistant Android version:**
1.3.1 (77)
**Android version:**
9 (Oxygen OS 9.0.9)
**Phone model:**
OnePlus 5
**Home Assistant version:**
0.104.0
**Last working Home Assistant release (if known):**

**Description of problem:**
It seems that as soon as I login with the Android App, the device is disabled and it doesn't "sync" if I don't enabled it:
![image](https://user-images.githubusercontent.com/9599441/72529299-4b0dae80-386d-11ea-985b-05762585d1af.png)

I did also notice some strange behaviour where I enabled it, but then I found it disabled again somehow.
The last bit seems to happen if I got back to the "homepage" and then back into the integration, while it "looks" still active if I press the "back" arrow which is part of the HA UI:
![image](https://user-images.githubusercontent.com/9599441/72529396-8b6d2c80-386d-11ea-8d47-4f1be14965cb.png)

So, basically it looks like it gets deactivated as soon as I leave the /config/ path

**Traceback (if applicable):**

```

```

**Additional information:**

**Comment by balloob** [comment]
That checkbox is to show/hide disabled entities. Has nothing to do with the actual entities.

**Comment by SeLLeRoNe** [comment]
I see, sorry about that then.
I tought that for disabled entites was the one "inside" the entity itself not that one.

My bad.
